cmd/collectors/unix/README.md

```diff
@@ -1,13 +1,21 @@
 
 
-# Unix Collector
+# Unix
 
-## Overview
+This collector polls resource usage by Harvest pollers on the local system. Collector might be extended in the future to monitor any local or remote process.
 
-This collector polls resource usage by Harvest pollers on the local system.
+### Table of Contents
```

Comment:
Do you think TOC is worth adding and more importantly, maintaining when GitHub automatically creates one for md files already?

Reply:
haven't noticed auto-generated TOC, but the TOC is not critical, what I suggest is to have a (non-strict,) uniform structure for all collectors, to make it easier to find the info you need. e.g. #Metric should provide what metrics the collector collects and how (if any) final values are calculated)

Reply:
The auto-TOC is near the top of the file
![image](https://user-images.githubusercontent.com/242252/120371110-5ab35900-c2e3-11eb-9f2d-78b6196781a2.png)